### ISSUE TITLE
Error: io.socket is undefined

### ISSUE THREAD

**Issue by peazz** [diegopamio/angular-sails-bind#20]
I am trying to integrate this with my angular project but as soon as I make a call inside a controller I get the following error:

```
 Error: io.socket is undefined
 _get@http://localhost:9000/bower_components/angular-sails-bind/angular-sails-bind.js:179:13
```

I have confirmed that sails.io.js is being included before angular etc etc, any ideas?

**Comment by eamador** [comment]
Same issue here too, did you manage to fix it?

**Comment by peazz** [comment]
Yeah I did, you need to manually include Sails.io.js before loading Angular Sails binds, the plug in does work it just not include the required file.

**Comment by diegopamio** [comment]
Yes, sorry for the long delay in answering, I've been really busy at may day-job.

You should include sails.io.js. It gets added by default on sailjs projects. I may need to add that to the readme.

**Comment by yura415** [comment]
Hi everyone. I think that I've had experienced this issue.
Here is how scripts included:
```html
  <script src="js/dependencies/sails.io.js"></script>
  <!-- build:js(src) scripts/vendor.js -->
  <!-- bower:js -->
......
  <script src="../bower_components/angular-sails-bind/dist/angular-sails-bind.js"></script>
.......
  <script src="../bower_components/angular-socket-io/socket.js"></script>
.......
  <!-- endbower -->
  <!-- endbuild -->
```
And here is my code:
```js
  /** @ngInject */
  function DataRowsController(..., $scope, $sailsBind, ...) {
...
    $sailsBind.bind("proxy", $scope);
...
  }
```

If I navigate to some route that not uses sails-bind (so sails.io will connect normally) and then navigate to route that uses it -- it works OK. But if I open app at route that uses sails-bind, it will fail with error:
![screen shot 2015-10-01 at 13 51 43](https://cloud.githubusercontent.com/assets/3534110/10218952/9c4f1ed6-6843-11e5-8419-b8ea56cb6eff.png)
